src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/PackageDetailsTabViewModel.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Shell;
+using NuGet.VisualStudio;
+using NuGet.VisualStudio.Internal.Contracts;
+
+namespace NuGet.PackageManagement.UI.ViewModels
+{
+    public class PackageDetailsTabViewModel : ViewModelBase, IDisposable
+    {
+        private bool _disposed = false;
+        private bool _readmeTabEnabled;
+
+        public ReadmePreviewViewModel ReadmePreviewViewModel { get; private set; }
+
+        public DetailControlModel DetailControlModel { get; private set; }
+
+        public ObservableCollection<TitledPageViewModelBase> Tabs { get; private set; }
```

Comment:
I believe last week in your feature branch PR I suggested making this an IReadOnlyList. Can you help me understand why this collection needs to be dynamic?

Reply:
Oh sorry, if I make the tab a ReadonlyList then I cannot make changes to the items in the tabs, they need to be instantiated in the constructor. Right now, I'm instantiating the VM in the constructor of the view, so I don't have all the components necessary to instantiate the VM. I think once we refactor the DetailControlModel it can handle creating the VM.